pinc/Project.inc

```diff
@@ -1124,6 +1105,13 @@ class Project
         return $page_image_names;
     }
 
+    public function validate_page_table_exists()
+    {
+        if (!$this->pages_table_exists) {
+            throw new NoProjectPageTable(_("Project page table does not exist."));
```

Comment:
It took me a while to figure out how to trigger this particular message. I mostly got "Page table does not exist". I was finally able to trigger it by going to the images index for an active project and subbing in a deleted projectid in the URL. Are there other situations that _**should**_ trigger the message that maybe aren't?

Reply:
It looks like the message "Page table does not exist" is only made in project_quick_check.
The only ways that  there cannot be a page table seem to be if the project is deleted or archived. Or perhaps a system crash during project creation.

Reply:
I may not have been clear enough here, either. What I wanted to know is if there are other places than the image index where the message in the code here ("Project page table does not exist.") should be seen. It's not in Project Quick Check, and I guess I wanted to make sure that it is not supposed to be.